app/controllers/keywords_controller.rb

```diff
@@ -4,10 +4,13 @@ class KeywordsController < ApplicationController
   include Pagy::Backend
 
   def index
-    pagy, keywords_list = pagy(keywords)
+    pagy, keywords_list = pagy(keywords_query.keywords_filtered)
 
     render locals: {
-      pagy: pagy, keywords: KeywordsCollectionPresenter.new(keywords_list),
+      pagy: pagy,
+      keywords: KeywordsCollectionPresenter.new(keywords_list),
```

Comment:
For the presenter class where we assign to a variable, we [rename](https://github.com/nimblehq/mia-web/tree/development/engines/mia_admin/app/controllers/mia_admin) it as `..._collection_presenter` or `presenter` . So how about do it same here?

Reply:
Ok, it makes sense. 
I will rename it in the FrontEnd branch #15 (PR - #85) as this will involve renaming locals in all views :)

Reply:
Resolved in [commit 
d884147](https://github.com/malparty/google-search-ruby/pull/85/commits/d884147abe2a77119148fbd8eba640c94e87ec71)